src/components/FruitCounter.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './FruitCounter.css'
+
+function FruitCounter({img, fruitName, setFruit, fruit}) {
+    return (
+        <section>
+            <h2>{img} {fruitName}</h2>
+            <button
+                type="button"
+                onClick={() => setFruit(fruit > 0 ? fruit - 1 : {fruit} = 0)}
+            >-
+            </button>
+            {fruit}
+            <button
+                type="button"
+                onClick={() => setFruit(fruit + 1)}
```

Comment:
Nu zal telkens wanneer `fruit` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setValue functie al krijgt:
```suggestion
                onClick={() => setFruit(currentValue => currentValue + 1)}
```